app_notifier/src/app_notifier/mail_notifier_plugin.py

```diff
@@ -17,6 +17,16 @@ def __init__(self):
 
     def app_manager_start_plugin(self, app, ctx, plugin_args):
         self.start_time = rospy.Time.now()
+        # Set mail title first for extensibility
+        self.use_app_start_time = False
+        if 'use_app_start_time' in plugin_args:
+            self.use_app_start_time = plugin_args['use_app_start_time']
+        if self.use_app_start_time:
+            mail_title = plugin_args['mail_title']
+            timestamp = '{0:%Y/%m/%d (%H:%M:%S)}'.format(
+                datetime.datetime.fromtimestamp(self.start_time.to_sec()))
+            mail_title += ': {}'.format(timestamp)
+            rospy.set_param('~mail_title', mail_title)
```

Comment:
I think it is good idea to share email title between many ROS nodes.
This can combined many emails into a single email thread.

One question is that which rosparam should be used to share the email title?
Your solution seems `~mail_title`, and another solution is `/email_topic/mail_title`.
(The latter may be more common for the other ROS nodes?)

Reply:
Thank you.
I changed `~mail_title` to `/email_topic/mail_title`[6634f0f](https://github.com/knorth55/app_manager_utils/pull/55/commits/6634f0f3db4d9edd304c75be7541dd45868eee7f)